### DISCUSSION TITLE
Can I do a dynamic website with Aleph.js?

### DISCUSSION THREAD

**Discussion by lcswillems** [alephjs/aleph.js#179] [Q&A] [answered]
On the documentation, it is written that Aleph.js can do SSR. For me, SSR means "server-side rendering", i.e. that the page is rendering on the server before being served. But `aleph build` always generate a static website. Am I missing something? Can I do a dynamic website with Aleph.js?

**Comment by hazae41** [comment, marked as answer]
You can do both a static and a dynamic website

Static : aleph build
Dynamic: aleph start

https://alephjs.org/docs/get-started
